examples/106-git-pipeline.py

```diff
@@ -6,13 +6,18 @@
      .describe("""This example of a pipeline using Git.
 
 The Git handler allows you to check your application source code into Git. Dataflow will checkout and build
-your code when the step starts.
+your code when the step starts. This example presents how one can use both go runtime & python runtime git steps.
 
 [Learn about Git steps](../docs/GIT.md)""")
      .step(
         (kafka('input-topic')
          .git('main', 'https://github.com/argoproj-labs/argo-dataflow', 'main', 'examples/git', 'golang:1.16',
               env=[{'name': "GOCACHE", 'value': "/tmp/.gocache"}], command=["go", "run", "."])
-         .kafka('output-topic')
+         .kafka('go-output-topic')
```

Comment:
Can we have a second separate Python example?

Reply:
As in `106-git-python-pipeline.py` that is a copy of the other one with a different runtime?